.github/pull_request_template.md

```diff
@@ -1,15 +1,19 @@
 <!--- Provide a general summary of your changes in the Title above -->
```

Comment:
For some reason, this file is formatted on commit. I've tried to roll it back, but it is formatted back again on commit.